pkg/model/gcemodel/autoscalinggroup.go

```diff
@@ -168,6 +168,16 @@ func (b *AutoscalingGroupModelBuilder) buildInstanceTemplate(c *fi.ModelBuilderC
 			//}
 			//t.Labels = labels
 
+			if ig.Spec.GuestAccelerators != nil && len(ig.Spec.GuestAccelerators) > 0 {
```

Comment:
This `if` can be removed. Setting this to empty slice allows unsetting it later if not wanted anymore. Otherwise, a `nil` value will be ignored.
```suggestion
```

Reply:
I have been coded in Java for many years with endless NullPointerExceptions :)

Reply:
Why setting and unsetting slice is better than a preventive check?

Reply:
tasks check actual vs model, when model is not `nil`.

Reply:
fixed